### ISSUE TITLE
Why not use dart:ffi?

### ISSUE THREAD

**Issue by MohiuddinM** [AdityaMulgundkar/flutter_opencv#6]
The android and ios versions are just wrappers over opencv c++ library. Won't it be beneficial if you can use dart:ffi to create bindings directly to c++.  That way you won't have to deal with writing platform specific codes, and at the same time make your library work with not just android and ios, but also on desktop platforms.

**Comment by flutter-clutter** [comment]
I agree, this is a good alternative. I used this technique in my project: https://github.com/flutter-clutter/flutter-simple-edge-detection -
however, it's only about edge detection. So if you want to implement the whole bindings using dart:ffi, feel free to have a look into my code and copy off the relevant parts @AdityaMulgundkar
